src/index.js

```diff
@@ -51,9 +51,9 @@ export default class FacebookTokenStrategy extends OAuth2Strategy {
    * @param {Object} options
    */
   authenticate(req, options) {
-    let accessToken = (req.body && req.body[this._accessTokenField]) || (req.query && req.query[this._accessTokenField]) || (req.headers && req.headers[this._accessTokenField]);
+    let accessToken = (req.body && req.body[this._accessTokenField]) || (req.query && req.query[this._accessTokenField]) || (FacebookTokenStrategy.parseAccessTokenHeader(req.headers, this._accessTokenField, this._authorizationField));
```

Comment:
If you made static method that parses access token then better to move there body and query too. Like `let accessToken = FacebookTokenStrategy.parseAccessToken(req)`. What you think?